content/chapter-03/Resilience.md

```diff
@@ -0,0 +1,15 @@
+# Resilience of software design
```

Comment:
Resilience of software design => Resilient Software Design

Reply:
Done